README.md

```diff
@@ -210,6 +210,40 @@ Since keys are unique there is no way to map values to keys using `Colors.value(
 Inheriting from a `Ruby::Enum` class, all defined enums in the parent class will be accessible in sub classes as well.
 Sub classes can also provide extra enums as usual.
 
+``` ruby
+class PrimaryColors
+  include Ruby::Enum
+
+  define :RED, 'RED'
+  define :GREEN, 'GREEN'
+  define :BLUE, 'BLUE'
+end
+
+class RainbowColors < PrimaryColors
```

Comment:
The example is super clear, this works. 

I do have a question. In the scenario with colors inheritance is the wrong object oriented pattern to use. Rainbow colors don't "inherit" primary colors, they are a combination of primary colors and other colors. In this case we would want a module and mixins. Should we allow `class PrimaryColors` be `module PrimaryColors`, `extend Ruby::Enum`, then `include PrimaryColors` into `RainbowColors` or something like that? ... for another PR.

Maybe there's a better OOO example for README that we can use in a separate update? WDYT?

Reply:
Yeah. Semantically, it makes more sense to `include` the `PrimaryColors` in the `RainbowColors` class instead of inheriting them. 

Though, I see this is a minor implementation detail. Defining an enum doesn't actually create an instance of that class: `Colors::RED` isn't an instance of `Ruby::Enum` or `Colors`. It's a `String` or `Symbol`. 

There's little difference between inheriting from a class and including a module other than the Ruby semantics. Both options include the base/included class in the class hierarchy.

Regardless, I think this could be an update for another PR.

Reply:
100%